src/readme.html

```diff
@@ -3,96 +3,96 @@
 </ol>
 
 <h2>Updating</h2>
 <h3>Using the Automatic Updater</h3>
-<p>If you are updating from version 2.7 or higher, you can use the automatic updater:</p>
+<p>If you are updating from beta version 1.0.0 or higher, you can use the automatic updater:</p>
 <ol>
 	<li>Open <span class="file"><a href="wp-admin/update-core.php">wp-admin/update-core.php</a></span> in your browser and follow the instructions.</li>
-	<li>You wanted more, perhaps? That&#8217;s it!</li>
+	<li>That&#8217;s it!</li>
 </ol>
 
 <h3>Updating Manually</h3>
+<p>Updating manually can be done via (S)FTP or via the admin panel of the web host (such as cPanel).</p>
 <ol>
-	<li>Before you update anything, make sure you have backup copies of any files you may have modified such as <code>index.php</code>.</li>
-	<li>Delete your old WordPress files, saving ones you&#8217;ve modified.</li>
-	<li>Upload the new files.</li>
-	<li>Point your browser to <span class="file"><a href="wp-admin/upgrade.php">/wp-admin/upgrade.php</a>.</span></li>
+	<li>Before anything, make sure you to backup the entire site as well as the database.</li>
+	<li>From the root of the WordPress install remove all files <strong>except</strong> `.htaccess`, `wp-config.php` and the entire `wp-content` folder. Also if there are any files present, such as google verification files, it is important to leave them too.</li>
+	<li>Remove the `wp-content` folder from the installation package, then upload the rest to the root.</li>
+	<li>Point the browser to <span class="file"><a href="wp-admin/upgrade.php">/wp-admin/upgrade.php</a></span> and follow the instructions.</li>
 </ol>
 
-<h2>Migrating from other systems</h2>
-<p>WordPress can <a href="https://codex.wordpress.org/Importing_Content">import from a number of systems</a>. First you need to get WordPress installed and working as described above, before using <a href="wp-admin/import.php" title="Import to WordPress">our import tools</a>.</p>
+<h2>Migrating from WordPress</h2>
+<p>With the ClassicPress migration plugin, it is very easy to migrate from WordPress to ClassicPress. Please follow the <a href="https://docs.classicpress.net/installing-classicpress/#migrate-classicpress">documentation</a> for step-by-step instructions.</p>
 
 <h2>System Requirements</h2>
 <ul>
-	<li><a href="https://secure.php.net/">PHP</a> version <strong>5.2.4</strong> or higher.</li>
-	<li><a href="https://www.mysql.com/">MySQL</a> version <strong>5.0</strong> or higher.</li>
+	<li><a href="https://secure.php.net/">PHP</a> version <strong>5.6</strong> or higher.</li>
+	<li><a href="https://www.mysql.com/">MySQL</a> version <strong>5.6</strong> or higher.</li>
+	<li><a href="https://letsencrypt.org/">HTTPS</a> support.</li>
```

Comment:
A HTTPS-enabled server is not required to use ClassicPress.

You're probably thinking of the fact that all _outgoing_ API connections must be made through HTTPS.  There's no need to mention this in the readme.  I have never seen a server that does not support this, but WordPress can run in such places so we added a check to the migration plugin.

Reply:
Yes, that was exactly my line of thinking, sorry for mixing it up and thanks for explaining it.

Reply:
Let's move this line below, to Recommendations.  It can say "Serve your site via HTTPS with a [free SSL certificate](https://letsencrypt.org/)".